client/cypress/integration/pages/flightEdit.spec.js

```diff
@@ -0,0 +1,50 @@
+describe("check flight page", () => {
+  const flightId = 6;
+
+  before(() => {
+    cy.seedDb();
+    cy.visit("/");
+    cy.loginNormalUser();
+    cy.visit(`/flug/${flightId}`);
+  });
+
+  it("Check presence and function of edit button", () => {
+    cy.get("#flight-details");
+    cy.get("button").contains("Flug bearbeiten").click();
+    cy.url().should("include", `/flug/${flightId}/bearbeiten`);
+  });
+
+  const oldGliderId = "cd25b974-1e30-4969-ba46-34990461990d";
+  const newGliderId = "8f48aa72-6ea0-477e-ae3c-e76fa99e7fb5";
+
+  // const oldFlightReport = "eius ullam omnis nesciunt amet dolorem";
+  const newFlightReport = "Foo";
+
+  it("check edit flight details", () => {
+    cy.get("#glider-select").should("have.value", oldGliderId);
+    cy.get("#glider-select")
+      .select(newGliderId)
+      .should("have.value", newGliderId);
+
+    // TODO: How to check for a fragment of text?
+    // cy.get("#flightReport").contains(oldFlightReport);
+    cy.get("#flightReport").clear().type(newFlightReport);
+
+    cy.get("#hikeAndFlyCheckbox").should("not.be.checked").check();
+    cy.get("#logbookCheckbox").should("not.be.checked").check();
+
+    // TODO: Check photos
+
+    cy.get("button").contains("Speichern").click();
+    cy.url().should("include", `/flug/${flightId}`);
+
+    // TODO: How to chain this?
+    cy.get("#cyFlightDetailsTable1").contains("td", "U-Turn Bodyguard");
```

Comment:
Das grundsätzliche Problem hier ist, dass das "contains" immer noch zur Selection gehört und keine Assertion ist.

Den Fehler mache ich leider auch gerne.

Eine richtige Assertion wäre z.B. "should" die kannst du dann auch beliebig mit .and() erweitern.

Hier sind das ja auch zwei verschiedene Elemente im DOM. Deshalb sehe ich da auch kein Chaining. Glaube auch nicht das Cypress das so vorsieht.